libpod/image/image.go

```diff
@@ -334,8 +334,36 @@ func (i *Image) TopLayer() string {
 // Remove an image; container removal for the image must be done
 // outside the context of images
 func (i *Image) Remove(force bool) error {
-	_, err := i.imageruntime.store.DeleteImage(i.ID(), true)
-	return err
+	parent, err := i.GetParent()
+	if err != nil {
+		return err
+	}
+	if _, err := i.imageruntime.store.DeleteImage(i.ID(), true); err != nil {
```

Comment:
Wait, should that bool be `force`? If not, why do we even have a `force`  bool as an argument?

Reply:
It doesn't look like it should be `force` - which leaves me the question of why we even have a force parameter. We probably should be checking for containers using the image first...

Reply:
The bool is not force, it says commit. The description says the commit param tells it to remove the images and layers associated with it. If it is false, it just lists the images and layers that would be removed.

Reply:
So the force bool actually does nothing here, I guess.

We should move the container-handling logic in here at some point so it does.

Reply:
Should I remove the force bool?

Reply:
Leave it, but add a TODO comment that it does nothing

Reply:
Okay, done!